src/quest_manager/admin.py

```diff
@@ -218,8 +289,10 @@ def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
                 parent_quest = Quest.objects.get(import_id=import_id)
 
                 self.generate_simple_prereqs(parent_quest, data_dict)
-
-                self.generate_campaign(parent_quest, data_dict)
+                if (data_dict.get('campaign_title') or data_dict.get('campaign_import_id')):
```

Comment:
can you remove the outer parenthesis? Why they there?  this isn't javascript or C++ =D